docs/ovms_quickstart.md

```diff
@@ -0,0 +1,169 @@
+# OpenVINO&trade; Model Server quickstart
+
+## Installing the Server
+
+The OpenVINO&trade; Model Server can be installed in three ways - using a prebuilt docker image (preffered way), by building image on your own and by building the application locally from sources. All approaches are described below.
+
+### Prebuilt containers
+
+A prebuilt container with OpenVINO&trade; Model Server is available for download on [dockerhub](https://hub.docker.com/r/intelaipg/openvino-model-server/). This image

```

Comment:
Wrong link to Docker Hub (old link to intelaipg). Please update to https://hub.docker.com/r/openvino/ubuntu18_model_server. Also need to say it is based on Ubuntu 18, no longer cleanlinux

Reply:
Corrected.